### ISSUE TITLE
egular expressions

### ISSUE THREAD

**Issue by AndreyZavarin** [sanniassin/react-input-mask#199]
More complex masks can be defined as an array of regular expressions and constant characters.

// Canadian postal code mask
const firstLetter = /(?!.*[DFIOQU])[A-VXY]/i;
const letter = /(?!.*[DFIOQU])[A-Z]/i;
const digit = /[0-9]/;
const mask = [firstLetter, digit, letter, " ", digit, letter, digit];
return <InputMask mask={mask} />;

**Comment by ErikGMatos** [comment]
I have also been testing all day today and it did not work with regex, but I realized now that I was looking at the version 3.0 documentation, but actually at the moment the stable version is version 2, I will test with version 2 now if it works.

**Comment by reintroducing** [comment]
https://github.com/sanniassin/react-input-mask/issues/197

**Comment by SomtoUgeh** [comment]
Please any updates on this issue? @reintroducing @ErikGMatos

**Comment by reintroducing** [comment]
@SomtoUgeh did you read the comments above?

**Comment by SomtoUgeh** [comment]
@reintroducing Yes I did .. I am still experiencing the same issue .. I asked to get updates as regards the test he said he was going to perform

**Comment by Multi-Thinker** [comment]
Still the same!

**Comment by AndrejGajdos** [comment]
+1

**Comment by priyankadevgan1992** [comment]
+1

**Comment by gfox-dev** [comment]
+1

**Comment by gfox-dev** [comment]
Maybe let's do a fork and fix it? cause this lib seems to be nice even in 2022
